CHANGELOG.md

```diff
@@ -2,8 +2,13 @@
 
 ## Unreleased
 
+### Features
+
 - Offline caching ([#208](https://github.com/getsentry/sentry-unity/pull/208))
 - Breadcrumb categories added ([#206](https://github.com/getsentry/sentry-unity/pull/206))
+- Bump Sentry .NET SDK 3.4.0 ([#214](https://github.com/getsentry/sentry-unity/pull/214))
+  - [changelog](https://github.com/getsentry/sentry-dotnet/blob/main/CHANGELOG.md#340)
+  - [diff](https://github.com/getsentry/sentry-dotnet/compare/3.3.4...3.4.0)
```

Comment:
@semuserable @bitsandfoxes what about this as a format for bumping the git submodule?
We can link to the changelog of this specific release, but also, the diff between the tag we were before and the tag we just moved to. Would really help tracking changes